src/harness/reference_models/pre_iap_filtering/pre_iap_filtering.py

```diff
@@ -36,16 +37,20 @@ def preIapReferenceModel(protected_entities, sas_uut_fad, sas_test_harness_fads)
       from SAS test harnesses.
   """    
 
-
   # Invoke Inter SAS duplicate grant purge list reference model
-  inter_sas_duplicate_grant.interSasDuplicateGrantPurgeReferenceModel\
-                                               (sas_uut_fad, sas_test_harness_fads)
+  inter_sas_duplicate_grant.interSasDuplicateGrantPurgeReferenceModel(sas_uut_fad,
+                                                          sas_test_harness_fads)
 
-  # TODO
   # Invoke PPA, EXZ, GWPZ, and FSS+GWBL purge list reference models
+  list_of_fss_neighboring_gwbl = pre_iap_util.getFssNeighboringGwbl(protected_entities['gwblRecords'],
```

Comment:
Is there a reason why this routine is called here and not inside the zonePurgeReferenceModel() ?? (rather than requiring that the FSS passed to that one are the filtered one being within 150km of GWBL).

I mean is there other occasion where you want to call the zonePurgeReferenceModel without calling first the getFssNeighboringGwbl? If not it would be more robust against mistake to include it inside.

Reply:
This   is implemented as per F2F document.     This was discussed with Kate and approved to be implemented in similar   manner